syntax = "proto3";

message MyMessage {
  int32 id   = 1;
  string name = 2;
}

service Forwarder {
  rpc ForwardMessage(MyMessage) returns (Ack) {}
}

message Ack {
  bool success = 1;
  string  info = 2;
}